x-pack/plugins/cases/public/components/all_cases/index.tsx

```diff
@@ -0,0 +1,607 @@
+import { useDeleteCases } from '../../containers/use_delete_cases';
+import { EuiBasicTableOnChange } from './types';
+import { Panel } from '../panel';
+import {
+  UtilityBar,
+  UtilityBarAction,
+  UtilityBarGroup,
+  UtilityBarSection,
+  UtilityBarText,
+} from '../utility_bar';
+import { getBulkItems } from '../bulk_actions';
+import { CaseHeaderPage } from '../case_header_page';
+import { ConfirmDeleteCaseModal } from '../confirm_delete_case';
+import { getActions } from './actions';
+import { CasesTableFilters } from './table_filters';
+import { useUpdateCases } from '../../containers/use_bulk_update_case';
+import { useGetActionLicense } from '../../containers/use_get_action_license';
+import { getActionLicenseError } from '../use_push_to_service/helpers';
+import { CaseCallOut } from '../callout';
+import { ConfigureCaseButton } from '../configure_cases/button';
+import { ERROR_PUSH_SERVICE_CALLOUT_TITLE } from '../use_push_to_service/translations';
+import { CaseDetailsHrefSchema, LinkButton } from '../../common/components/links';
+import { Stats } from '../status';
+import { SELECTABLE_MESSAGE_COLLECTIONS } from '../../common/translations';
+import { getExpandedRowMap } from './expanded_row';
+import { isSelectedCasesIncludeCollections } from './helpers';
+
+const Div = styled.div`
+  margin-top: ${({ theme }) => theme.eui.paddingSizes.m};
+`;
+
+const FlexItemDivider = styled(EuiFlexItem)`
+  ${({ theme }) => css`
+    .euiFlexGroup--gutterMedium > &.euiFlexItem {
+      border-right: ${theme.eui.euiBorderThin};
+      padding-right: ${theme.eui.euiSize};
+      margin-right: ${theme.eui.euiSize};
+    }
+  `}
+`;
+
+const ProgressLoader = styled(EuiProgress)`
+  ${({ theme }) => css`
+    top: 2px;
+    border-radius: ${theme.eui.euiBorderRadius};
+    z-index: ${theme.eui.euiZHeader};
+  `}
+`;
+
+const getSortField = (field: string): SortFieldCase => {
+  if (field === SortFieldCase.createdAt) {
+    return SortFieldCase.createdAt;
+  } else if (field === SortFieldCase.closedAt) {
+    return SortFieldCase.closedAt;
+  }
+  return SortFieldCase.createdAt;
+};
+
+const EuiBasicTable: any = _EuiBasicTable;
+const BasicTable = styled(EuiBasicTable)`
+  ${({ theme }) => `
+    .euiTableRow-isExpandedRow.euiTableRow-isSelectable .euiTableCellContent {
+      padding: 8px 0 8px 32px;
+    }
+
+    &.isModal .euiTableRow.isDisabled {
+      cursor: not-allowed;
+      background-color: ${theme.eui.euiTableHoverClickableColor};
+    }
+
+    &.isModal .euiTableRow.euiTableRow-isExpandedRow .euiTableRowCell,
+    &.isModal .euiTableRow.euiTableRow-isExpandedRow:hover {
+      background-color: transparent;
+    }
+
+    &.isModal .euiTableRow.euiTableRow-isExpandedRow {
+      .subCase:hover {
+        background-color: ${theme.eui.euiTableHoverClickableColor};
+      }
+    }
+  `}
+`;
+BasicTable.displayName = 'BasicTable';
+
+export interface AllCasesProps {
+  configureCasesHref: string;
```

Comment:
nit, but i'd love to see long prop lists alphabetized 🔠

Reply:
👍🏾